src/main/java/org/traccar/protocol/TeltonikaProtocolDecoder.java

```diff
@@ -236,15 +236,7 @@ private static void register(int id, Set<String> models, BiConsumer<Position, By
             }
         });
         register(80, fmbXXX, (p, b) -> p.set("dataMode", b.readUnsignedByte()));
-        register(81, fmbXXX, (p, b) -> p.set(Position.KEY_OBD_SPEED, b.readUnsignedByte()));
-        register(82, fmbXXX, (p, b) -> p.set(Position.KEY_THROTTLE, b.readUnsignedByte()));
```

Comment:
Sure:
1. OBD Speed is AVL 37. This is natively reported by the FMBXXX devices without the need for CAN reader. Hence the conflict.
![image](https://github.com/traccar/traccar/assets/59513748/f8cecf87-53b5-4b54-8b80-9fcc901fc7e6)

2. Throttle is AVL. 41. This is also natively reported by the FMBXXX devices without the need for CAN reader. Hence the conflict.
![image](https://github.com/traccar/traccar/assets/59513748/90c3ab16-0156-478c-a09c-028975fd02ad)

Reply:
No, the devices report both - that's the idea of the different AVL ids. This code forces the assignment of the Throttle and  speed keys to the CAN AVLs.

Reply:
We are talking about different things - devices can report one or the other, or both. 
The problem is if we wish to have AVL 37 for throttle, the decoder assigns the CAN AVL.

Also, another thing that I just crossed my mind, the CAN adapters are not exclusively used with FMBXXX devices only.

Reply:
Here's the disadvantage to your suggestion - if we want to receive both AVL 37 and AVL 82 (KEY_THROTTLE) in the parameters we will see:
`io37: io37value, throttle: io82value`

Then if, we wish to use `io37` for throttle, then we cannot keep both values using computed attributes because we will end up with this in the positions table:
`throttle: io82value`

Hence the conflict. I don't know how to explain it more clearly.

Reply:
This is the first time you mentioned that someone would need both. Why that would be the case? And it also seems like a very corner case that vast majority of people don't care about.

Reply:
I cannot decide for what people might need or not need. Depending on the vehicle and reported data people might need both AVLs. 
I see a conflict (or for the very least unclean code) and I point it out. That's all.

Reply:
> I cannot decide for what people might need or not need. Depending on the vehicle and reported data people might need both AVLs. I see a conflict (or for the very least unclean code) and I point it out. That's all.

Ok, so we have device which plugs directly to obd and device which uses lv_can.. shame for teltonka that for same functions it will have different avl's don't think ever it will be used in both cases.. so you use one or another..

Reply:
Well, actually LV-CAN is a separate addon device. It's like an enhancement and it can be used both with FMBXXX and non FMBXXX devices.
So it makes sense that the standalone FMBXXX series send the throttle and so do the LV-CAN addon (with a different ID so that we know which is which).

Reply:
> Well, actually LV-CAN is a separate addon device. It's like an enhancement and it can be used both with FMBXXX and non FMBXXX devices. So it makes sense that the standalone FMBXXX series send the throttle and so do the LV-CAN addon (with a different ID so that we know which is which).

Yes,also you can plug Bluetooth obd and connect to teltonika.. its third source..